### ISSUE TITLE
Clipboard or JS Bug

### ISSUE THREAD

**Issue by SpringsTea** [zenorocha/clipboard.js#186]
I had an issue where only the first time something was copied to clipboard would work, and all subsequent copies would keep pasting the original value. I was using this code:

```javascript
new Clipboard(".copy-data", {
      text: function(trigger) {
        console.log(trigger);//1
        console.log($(trigger).data('clipboard-text'));//2
        console.log('end');
        return $(trigger).data('clipboard-text'); 
      }
    });
```

So triggering this 3 times gives me the following output:

class="clickable copy-data" data-clipboard-text="35924">Copy Ticket ID#
35924
end

class="clickable copy-data" data-clipboard-text="35925">Copy Ticket ID#
35924
end

class="clickable copy-data" data-clipboard-text="35926">Copy Ticket ID#
35924
end

Basically, the trigger is showing the correct data, but when I retrieve the attribute, it's incorrect.
I'm not really sure if this is related to clipboard, or just JS.

**Comment by zenorocha** [comment]
Please share a JSFiddle so we can debug it properly.
